action.yml

```diff
@@ -1,78 +1,79 @@
 name: Accessibility alt text bot
-description: 'This action will check a repos issue, discussion, or PR for correct alt text usage.'
+description: "This action will check a repos issue, discussion, or PR for correct alt text usage."
 branding:
-  icon: 'eye'
-  color: 'purple'
+  icon: "eye"
+  color: "purple"
 runs:
   using: "composite"
   steps:
     - name: Runs alt text check and adds comment
       run: |
-          source ${{ github.action_path }}/flag-alt-text.sh
-          source ${{ github.action_path }}/queries.sh
+        source ${{ github.action_path }}/flag-alt-text.sh
```

Comment:
Looks like we can delete this given the file has been removed!

Reply:
I am testing this branch in one of my repos! ([Related workflow error](https://github.com/khiga8/link-checker/actions/runs/6487639086/job/17618370808))